### PR TITLE
if we can't render don't bother

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@regro/auto-tick ready for review. This will unstick some py38 migrations

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -98,7 +98,10 @@ class LicenseMigrator(MiniMigrator):
 
     def migrate(self, recipe_dir, attrs, **kwargs):
         # Use conda build to do all the downloading/extracting bits
-        md = render(recipe_dir, config=Config(**CB_CONFIG))[0][0]
+        md = render(recipe_dir, config=Config(**CB_CONFIG))
+        if not md:
+            return
```
[the remark this answers is not in the document]
Comment (CJ-Wright) [reply conda_forge_tick/migrators.xsh]: Not currently. Do you want that in this PR?

[the remark this answers is not in the document]
Comment (ocefpaf) [reply conda_forge_tick/migrators.xsh]: Nope. Just curious. This is good to go IMO.

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**